components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -0,0 +1,70 @@
+//
+//  FeatureHighlightTitleBodyAccessibilityMutatorTests.m
+//  MaterialComponentsUnitTests
+//
+//  Created by Mohammad Cazi on 3/5/18.
+//
+
+#import <XCTest/XCTest.h>
+#import "MaterialFeatureHighlight.h"
+#import "MDCFeatureHighlightAccessibilityMutator.h"
+
+static NSArray<UIColor *> *testColors(){
+  return @[[UIColor whiteColor], [UIColor blackColor], [UIColor redColor], [UIColor orangeColor],
+           [UIColor greenColor], [UIColor blueColor], [UIColor grayColor]];
+}
+
+@interface FeatureHighlightTitleBodyAccessibilityMutatorTests : XCTestCase
+
+@end
+
+@implementation FeatureHighlightTitleBodyAccessibilityMutatorTests
+
+- (void)testMutateChangesTextColor {
+  for (UIColor *color in testColors()) {
+    // Given
+    MDCFeatureHighlightViewController *highlightVC =
+        [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:[UIView new]
+                                                                completion:nil];
+    MDCFeatureHighlightView *highlightView = (MDCFeatureHighlightView *)highlightVC.view;
+
+    // Making the background color the same as the title/body color.
+    highlightView.outerHighlightColor = color;
+    highlightView.titleColor = color;
+    highlightView.bodyColor = color;
+
+    // When
+    [MDCFeatureHighlightAccessibilityMutator
+        changeTitleAndBodyColorForFeatureHighlightViewControllerIfApplicable:highlightVC];
+
+    // Then
+    XCTAssertNotEqualObjects(highlightView.titleColor, color, @"Not same color");
```

Comment:
Please assert the correct colors here.    You can use code similar to `UIColor+ForcedFloatComparison.swift` to extract the expected values for the test.

Reply:
hmm I don't think, this should depend on the color it self since accessibility could possibly change the color at any time. It would be ideal to make sure that this class is setting the color to what accessibility returns but without OCMock that requires a lot of effort.

Reply:
That would be the point - if MDFTextAccessibility changes the colors, we should flag that as a change to be addressed.  Automatic unit tests are the easiest way to get that review noticed.